neurokit2/signal/signal_fixpeaks.py

```diff
@@ -225,8 +225,8 @@ def _find_artifacts(peaks, c1=0.13, c2=0.17, alpha=5.2, window_width=91,
         if np.abs(drrs[i]) <= 1:    # Figure 1
             i += 1
             continue
-        eq1 = np.logical_and(drrs[i] > 1, s12[i] < (-c1 * drrs[i] - c2))    # Figure 2a
-        eq2 = np.logical_and(drrs[i] < -1, s12[i] > (-c1 * drrs[i] + c2))    # Figure 2a
+        eq1 = np.logical_and(drrs[i] > 1, s12[i] < (-c1 * drrs[i] + c2))    # Figure 2a
```

Comment:
pinging @JanCBrammer

Reply:
Thank you @JanCBrammer! 
Those equations make sense with Fig2a indeed ( I was careless and only following the decision tree) 👍